.github/workflows/orch-tester-dockerimage.yml

```diff
@@ -0,0 +1,29 @@
+name: Orch Tester Docker Image CI
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Docker Registry
+      env:
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      run: docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+
+    - name: Tags
+      id: tags
+      uses: livepeer/action-gh-release-tags@v0
```

Comment:
Some of the other workflows have this:
```yaml
      with:
        always-latest-on-branch: master
```

Do you think it'd be worth it here as well? Otherwise, it will only be tagged `latest` on dockerhub when you explicitly create a semver tag with a stable version (that might be desired as well)

Reply:
Yeah, good idea. Added in 64379c7c3d09e000596f0eee6dc69816c9239596

I see `mist-api-connector-dockerimage.yml` don't have `always-latest-on-branch: master`. Do you think it might be worth to add it there as well? @victorges